src/viam/sdk/module/service.cpp

```diff
@@ -252,13 +250,8 @@ ModuleService::~ModuleService() {
 }
 
 void ModuleService::serve() {
-    const mode_t old_mask = umask(0077);
```

Comment:
I forgot to mention this in my batch of comments. This code, which has been here forever, has always done nothing. It sure _looked_ like it was making the socket that we were going to use for communication with RDK, but look carefully and you can see that we never actually bind the socket anywhere, or invoke it in any way. The actual socket gets made by gRPC.